tuf/utils/utils.go

```diff
@@ -146,3 +147,15 @@ func FindRoleIndex(rs []*data.Role, name string) int {
 	}
 	return -1
 }
+
+// URLFilePath generates the appropriate HTTP URL path for the targets file,
```

Comment:
I think this is used for more than just the targets file -- it's any metadata we have a checksum for right?  We should update the comment if so